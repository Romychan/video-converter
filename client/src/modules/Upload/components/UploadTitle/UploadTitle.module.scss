.header {
  margin-bottom: 4rem;
  text-align: center;
}

.title {
  font-size: 1.3125rem;
  margin-bottom: 1rem;
}
