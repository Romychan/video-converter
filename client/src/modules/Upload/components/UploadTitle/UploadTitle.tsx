import clsx from 'clsx';

import styles from './UploadTitle.module.scss';

export const UploadTitle = () => {
  return (
    <div className={styles.header}>
      <h1 className={clsx(styles.title, 'subtitle')}>Конвертер видео</h1>
      <p className={clsx(styles.text, 'text')}>
        Video Converter - это онлайн-конвертер видео, который позволяет
        конвертировать видеофайлы в таких форматах, как MP4, AVI, GIF, MOV и
        другие.
      </p>
    </div>
  );
};
