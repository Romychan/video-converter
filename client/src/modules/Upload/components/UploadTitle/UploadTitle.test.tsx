import { RenderResult, render, screen } from '@testing-library/react';

import { UploadTitle } from './UploadTitle';

describe('UploadTitle', () => {
  let container: RenderResult;

  beforeEach(() => {
    container = render(<UploadTitle />);
  });

  it('should render correctly', () => {
    expect(screen.getByText(/Конвертер видео/)).toBeInTheDocument();
  });

  it('should match snapshot', () => {
    expect(container).toMatchSnapshot();
  });
});
