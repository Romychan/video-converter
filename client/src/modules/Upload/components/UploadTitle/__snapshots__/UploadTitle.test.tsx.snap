// Vitest Snapshot v1

exports[`UploadTitle > should match snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="_header_6c7c0d"
      >
        <h1
          class="_title_6c7c0d subtitle"
        >
          Конвертер видео
        </h1>
        <p
          class="_text_6c7c0d text"
        >
          Video Converter - это онлайн-конвертер видео, который позволяет конвертировать видеофайлы в таких форматах, как MP4, AVI, GIF, MOV и другие.
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="_header_6c7c0d"
    >
      <h1
        class="_title_6c7c0d subtitle"
      >
        Конвертер видео
      </h1>
      <p
        class="_text_6c7c0d text"
      >
        Video Converter - это онлайн-конвертер видео, который позволяет конвертировать видеофайлы в таких форматах, как MP4, AVI, GIF, MOV и другие.
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
